extensions/redis-client/runtime/src/main/java/io/quarkus/redis/client/runtime/RedisClientImpl.java

```diff
@@ -5,15 +5,19 @@
 
 import io.quarkus.redis.client.RedisClient;
 import io.smallrye.mutiny.Uni;
+import io.vertx.mutiny.redis.client.Command;
 import io.vertx.mutiny.redis.client.RedisAPI;
+import io.vertx.redis.client.Request;
 import io.vertx.redis.client.Response;
 
 class RedisClientImpl implements RedisClient {
     private final RedisAPI redisAPI;
+    private final MutinyRedis mutinyRedis;
     private final Duration timeout;
 
-    public RedisClientImpl(RedisAPI redisAPI, Duration timeout) {
+    public RedisClientImpl(RedisAPI redisAPI, MutinyRedis mutinyRedis, Duration timeout) {
```

Comment:
`io.quarkus.redis.client.runtime.MutinyRedis` this API is in runtime package, I am wondering if in the documentation we can use the `io.vertx.mutiny.redis.client.Redis` which IIRC can be injected based on  https://github.com/quarkusio/quarkus/blob/1064752e4e19f17edaddd0e5d73902f83aa36308/extensions/redis-client/deployment/src/main/java/io/quarkus/redis/client/deployment/RedisClientProcessor.java#L141